portfolio/src/main/webapp/index.html

```diff
@@ -138,9 +135,21 @@ <h2>Misc</h2>
     <div class="comment_background">
       <div id="comments_sec">
         <h2>Leave a Comment!</h2>
-        <form action="/comments" method="POST">
+        <form id="cmt_form" action="/comments" method="POST">
+          <h5>
```

Comment:
This should probably be a [`<label>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). They're especially helpful for users of screenreaders.

Reply:
Ok